scripts/fix-shrinkwrap.js

```diff
@@ -52,36 +62,38 @@ function fixup(dependency) {
 
   return false;
 
-}
+};
 
-// Walk the shrinkwrap dependency tree,
-// and fix up any mangled versions along the way
-// while counting how many have been fixed (because, why not?)
-function walk(tree) {
+/**
+ * Walk the shrinkwrap dependency tree,
+ * and fix up any mangled versions along the way
+ * while counting how many have been fixed (because, why not?)
+ * @param {Object} tree - dependency tree
+ * @returns {Number}
+ */
+const walk = (tree) => {
 
   if (!tree.dependencies) {
-    return 0; // Nothing to do here
+    // Nothing to do here
+    return ZERO;
   }
 
-  var keys = Object.keys(tree.dependencies);
-  var fixes = 0;
-  var dep = null;
+  let fixes = ZERO;
 
-  for (var i = 0; i < keys.length; i++) {
-    dep = tree.dependencies[keys[i]];
-    fixes += fixup(dep) ? 1 : 0;
+  _.forEach(tree.dependencies, (dep) => {
+    fixes += fixup(dep) ? ONE : ZERO;
     fixes += walk(dep);
-  }
+  });
 
   return fixes;
 
-}
+};
 
 const fixes = walk(shrinkwrap);
 
 // Save the shrinkwrap back
-fs.writeFileSync(SHRINKWRAP_FILE, JSON.stringify(shrinkwrap, null, 2) + '\n');
+fs.writeFileSync(SHRINKWRAP_FILE, `${JSON.stringify(shrinkwrap, null, TWO)}\n`);
 
-time = (Date.now() - time) / 1000;
+time = (Date.now() - time) / THOUSAND;
```

Comment:
In keeping with the style of the rest of the codebase, `THOUSAND` here should probably be something like `MILLISECONDS_IN_A_SECOND` ?